main.cpp

```diff
@@ -84,43 +106,78 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
-
+int makeMathOperation(int firstOperand, int secondOperand, int operationType = 0)
+{
+    ignoreUnused(firstOperand, secondOperand, operationType);
+    return {};
+}
 /*
  2)
  */
-
+bool bookARoom(int numberOfDays, int numberOfBeds = 1)
+{
+    ignoreUnused(numberOfDays, numberOfBeds);
+    return {};
+}
 /*
  3)
  */
-
+bool isDressSizeAvailable(int typeOfDress, char dressSize = 'S' ) 
+{
+    ignoreUnused(typeOfDress, dressSize);
+    return {};  
+}
 /*
  4)
  */
-
+int numOfMangaAvailable(char initialLetterOfTitle ,int typeOfManga = 2)
+{
+    ignoreUnused(initialLetterOfTitle, typeOfManga);
+    return {};
+}
 /*
  5)
  */
-
+void increaseVolume(float volumeValue, float gainValue = 1.0f)
+{
+    ignoreUnused(volumeValue, gainValue);
+}
 /*
  6)
  */
-
+void addFiltersToChain(int numOfFilters, char filterCutType = 'l', float gainValue = 1.0f)
+{
+    ignoreUnused(numOfFilters, filterCutType, gainValue);
+}
 /*
  7)
  */
-
+void orderBeer(int debitCardNumber, int numberOfBeers = 1)
+{
+    ignoreUnused(debitCardNumber, numberOfBeers);
+}
 /*
  8)
  */
-
+void resumeSong(int momentTimeStamp, int sampleRate = 44100)
+{
+    ignoreUnused(momentTimeStamp, sampleRate);
+}
 /*
  9)
  */
-
+float OfferToAuction(float actualOffer, float yourOffer = 50.0f)
```

Comment:
FIXME: function names should start with a lowercase letter.  See Readme.MD for more details
- [ ] revise